cartoframes/io/carto.py

```diff
@@ -89,6 +91,9 @@ def to_carto(dataframe, table_name, credentials=None, if_exists='fail', geom_col
             uses the name of the index from the dataframe.
         cartodbfy (bool, optional): convert the table to CARTO format. Default True. More info
             `here <https://carto.com/developers/sql-api/guides/creating-tables/#create-tables>`.
+        ignore_quota_warning (bool, optional): ignore the warning of the possible quota exceeded
```

Comment:
I think we can use `skip` instead of `ignore` in the param name. What do you think?